src/icons/fileIcons.ts

```diff
@@ -116,6 +116,10 @@ export const fileIcons: FileIcons = {
         'tga',
         'xcf',
         'jfif',
+        'ppm',
+        'pbm',
```

Comment:
Please remove this line.

Reply:
You mean replace with **ppm**. Don't you?

Reply:
Yes, just remove "pbm" from this list please. It's already in it somewhere above and no need to have it twice.